addons/account/static/src/js/account_followup_report_widgets.js

```diff
@@ -0,0 +1,168 @@
+openerp.account.FollowupReportWidgets = openerp.account.ReportWidgets.extend({
+    events: _.defaults({
+        'click .change_exp_date': 'displayExpNoteModal',
+        'click #savePaymentDate': 'changeExpDate',
+        'click .followup-email': 'sendFollowupEmail',
+        'click .followup-letter': 'printFollowupLetter',
+        'click .followup-skip': 'skipPartner',
+        "change *[name='blocked']": 'onChangeBlocked',
+        'click .oe-account-set-next-action': 'setNextAction',
+        'click #saveNextAction': 'saveNextAction',
+        'click .oe-account-followup-one-week': 'setNextAction',
+        'click .oe-account-followup-two-weeks': 'setNextAction',
+        'click .oe-account-followup-one-month': 'setNextAction',
+        'click .oe-account-followup-two-months': 'setNextAction',
+    }, openerp.account.ReportWidgets.prototype.events),
+    saveNextAction: function(e) {
+        e.stopPropagation();
+        e.preventDefault();
+        var note = $("#nextActionNote").val().replace(/\r?\n/g, '<br />').replace(/\s+/g, ' ');
+        var target_id = $("#nextActionModal #target_id").val();
+        var date = $("#nextActionDate").val();
+        date = openerp.web.parse_value(date, {type:'date'})
+        var contextModel = new openerp.Model('account.report.context.followup');
+        return contextModel.call('change_next_action', [[parseInt(target_id)], date, note]).then(function (result) {
+            $('#nextActionModal').modal('hide');
+            $('div.page.' + target_id).find('.oe-account-next-action').html(openerp.qweb.render("nextActionDate", {'note': note, 'date': date}));
+        });
+    },
+    setNextAction: function(e) {
+        e.stopPropagation();
+        e.preventDefault();
+        var target_id = $(e.target).parents("div.page").attr("class").split(/\s+/)[3];
```

Comment:
xmo will eat you if you continue to do things like that. You can NEVER base yourselve on the position of the class elements since sooner or later someone will add classes to redesign it. If you want to see if the element contain a specific class you can use .hasClass() method
ex: $(e.target).hasClass('oe-account-followup-one-week')
If you need to get some information like target_id that is not always the same, don't put it inside the class but inside another attributes. All browser support custom data attribute if prefixed by "data-" you can then access those information in js using $(selector_here).data('target-id') (without the preffixed data- here) or $(selector_here).attr('data-target-id')
ex: <div class="my_class" data-target-id="my_target" data-smth="another_info">